app/src/main/java/de/danoeh/antennapod/fragment/FeedSettingsFragment.java

```diff
@@ -114,6 +116,43 @@ private void updateAutoDeleteSummary() {
         }
     }
 
+    private void setupVolumeReductionPreferences() {
+        ListPreference volumeReductionPreference = (ListPreference) findPreference("volumeReduction");
+        volumeReductionPreference.setOnPreferenceChangeListener((preference, newValue) -> {
+            switch ((String) newValue) {
+                case "off":
+                    feedPreferences.setVolumeReductionSetting(FeedPreferences.VolumeReductionSetting.OFF);
+                    break;
+                case "light":
+                    feedPreferences.setVolumeReductionSetting(FeedPreferences.VolumeReductionSetting.LIGHT);
+                    break;
+                case "heavy":
+                    feedPreferences.setVolumeReductionSetting(FeedPreferences.VolumeReductionSetting.HEAVY);
+                    break;
+            }
+            feed.savePreferences();
+            updateVolumeReductionSummary();
+            // TODO maxbechtold Check if we can call setVolume for the PlaybackService, if running. Else, show toast?
+            return false;
+        });
+    }
+
+    private void updateVolumeReductionSummary() {
```

Comment:
This method updates the value, not the summary. Please rename

Reply:
Thanks